evap/results/templates/distribution_widget.html

```diff
@@ -2,13 +2,16 @@
 {% load evaluation_filters %}
 
 {% spaceless %}
-{% if average is not None %}
-    <div class="d-flex">
-        {% include "distribution_with_grade.html" with distribution=distribution average=average weight=weight weight_sum=weight_sum %}
+{% if course_or_evaluation.avg_grade is not None %}
+    <div class="d-flex"
+        data-toggle="tooltip" data-placement="left" data-fallback-placement='["left", "bottom"]'
+        title="{% spaceless %}{% include 'result_widget_tooltip.html' with question_result=course_or_evaluation.single_result_rating_result weight_info=course_or_evaluation|weight_info %}{% endspaceless %}"
```

Comment:
Any idea for a nicer solution than to put `{% spaceless %}` around the tooltip? The problem lies in courses or evaluations with no weighting information that the tooltip is considered empty, but the included file contains whitespaces (even if I wrap the entire file into `{% spaceless %}`, one line break after the closing tag persists).

Reply:
i don't mind having `spaceless` here.